README.md

```diff
@@ -42,7 +42,7 @@ Clone the gazebo plugins repository to your computer. IMPORTANT: If you do not c
 ```bash
 mkdir -p ~/src
 cd src
-git clone https://github.com/Dronecode/sitl_gazebo.git
+git clone https://github.com/PX4/sitl_gazebo.git
```

Comment:
`git clone --recursive https://github.com/PX4/sitl_gazebo.git`. That way, you avoid adding https://github.com/PX4/sitl_gazebo/pull/211/files#diff-04c6e90faac2675aa89e2176d2eec7d8R76